test/integration/webhookgithub_test.go

```diff
@@ -327,6 +327,42 @@ func mockBuildConfigImageParms(imageName, imageStream, imageTag string) *buildap
 						Secret: "secret102",
 					},
 				},
+				{
+					Type: buildapi.GitLabWebHookBuildTriggerType,
+					GitLabWebHook: &buildapi.WebHookTrigger{
```

Comment:
I definitely agree. In another PR, I had to make a webhookgeneric_test, which doesn't look good. On it.